angular/projects/common/src/app/constants/auth.ts

```diff
@@ -0,0 +1,6 @@
+export const ROLES = [
```

Comment:
These already exist in common/app/utils record-constants.ts. There's a class there called ApplicationRoles that's used in the UI. Can probably remove that one (or use that one) rather than having it in two places.

Reply:
Good catch - might be better to move it out from record constants

Reply:
And in api/src/uils/constants/misc.js...

Reply:
Well, it's nice to have options!